docs/guide/beyond-angularfire.md

```diff
@@ -77,10 +72,10 @@ or you can host it at any custom domain on one of our paid plans. Check out
 
 There are many additional resources for learning about using Firebase with Angular applications:
 
-* Browse the [AngularFire API documentation](https://angularfire.firebaseapp.com/api.html).
+* Browse the [AngularFire API reference](/docs/reference.md).
```

Comment:
This is correct - we've doubled checked all these links, Github is smart enough to handle the repo-specific /